### ISSUE TITLE
OS Package Requests

### ISSUE THREAD

**Issue by MCMrARM** [minecraft-linux/mcpelauncher-manifest#25]
In this issue, please report for what OSes you would like to have an official prebuilt package for.

If an OS has already been noted in a comment, please upvote the comment instead of posting a new one.

Currently approved OSes:
- Ubuntu 18.04
- Ubuntu 16.04
- Arch (AUR)

**Comment by Mhowser** [comment]
Ubuntu 18.04

**Comment by MCMrARM** [comment]
Yeah there is an OS X package. I meant Linux distributions here though. Ubuntu 18.04 is already approved.

**Comment by Austcool-Walker** [comment]
@MCMrARM a manjaro repo would be kinda nice

**Comment by MCMrARM** [comment]
Does Manjaro not support AUR?

**Comment by mikroskeem** [comment]
Arch Linux repository would be generally great (not only Manjaro), because saves time and resources. AUR package could stay as-is to test the bleeding edge maybe? :smile:

**Comment by aFoxNamedMorris** [comment]
Ubuntu 18.10

**Comment by armandopartida0** [comment]
Solus OS

**Comment by caoli5288** [comment]
Sugguest generic rpm pack. Can built under openbuildservice.

**Comment by suckybell** [comment]
A universal Linux package would be great.  Flatpak and AppImage work the best in my experience.  They work on pretty much any distro.

**Comment by jv110** [comment]
Debian

**Comment by No3Mc** [comment]
Kali Linux
